app/src/main/java/com/github/dedis/student20_pop/utility/network/IMessageListener.java

```diff
@@ -0,0 +1,14 @@
+package com.github.dedis.student20_pop.utility.network;
+
+/**
+ * Listener of received messages by the websocket
+ */
+public interface IMessageListener {
```

Comment:
Just by curiosity, why is it called IMessageListnener (why the I) ?

Reply:
For Interface, in some convention, we always add an 'I' before an interface name. I am too used to this convention.
I'm changing it for concistency